tools/coverage.sh

```diff
@@ -10,48 +13,61 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ################################################################################
-
-#!/bin/sh
 #
 # Usage
 #
 #   COVERAGE_CPUS=32 tools/coverage.sh [/path/to/report-directory/] [targets]
 #
 # COVERAGE_CPUS defaults to 2, and the default destination is a temp
 # dir.
-set -e
-genhtml=$(which genhtml)
-if [[ -z "${genhtml}" ]]; then
+
+set -eu
+
+genhtml=$(command -v genhtml)
+if [ -z "${genhtml}" ]; then
```

Comment:
[[ is preferred over [.

https://google.github.io/styleguide/shell.xml?showone=Test,_%5B_and_%5B%5B#Test,_%5B_and_%5B%5B